rpc/netmap.go

```diff
@@ -44,3 +45,19 @@ func NetworkInfo(
 
 	return resp, nil
 }
+
+// Snapshot executes NetmapService.NetmapSnapshot RPC.
+func Snapshot(
```

Comment:
`rpc.Snapshot` seems too generic, may be `rpc.NetmapSnapshot`?

Reply:
Oh, forgot it isn't a `netmap` package, fixed.